man/signal-cli-dbus.5.adoc

```diff
@@ -198,6 +199,19 @@ MessageReceived(timestamp<x>, sender<s>, groupId<ay>, message<s>, attachments<as
 
 This signal is received whenever we get a private message or a message is posted in a group we are an active member
 
+== Examples
+
+Send a text message (without attachment) to a contact::
+dbus-send --print-reply --type=method_call --dest="org.asamk.Signal"
```

Comment:
I think the command should be in a single line

Reply:
Battling vim's auto-formatting :)